content/versions-comparison/Serenity-news.json

```
@@ -14,7 +14,7 @@
     "link": "/pim/serenity/updates/2020-07.html#new-create-and-update-rules-from-the-pim"
   },
   {
-    "title":"Quantities in product association 🔗",
+    "title":"Association with quantities 🔗",
```

```suggestion
    "title":"Product association with quantities 🔗",
```
Seems important to state it, as people may not know that we provide associations between products.